src/providers/provider.ts

```diff
@@ -14,6 +17,7 @@ export class Provider {
   protected _eventManager: EventManager;
   protected _logger: Logger;
   public cuePointManager: CuePointManager | null = null;
+  protected _cuePointsData: any[] = [];
```

Comment:
correct me if I'm wrong- seems like there is no use in `_cuePointsData` in this file. it is only defined and being reset in destroy..
i believe that `_cuePointsData` is now the `_pendingCuePointsData` in vod-provider file.
If so, you can remove it from this file.